### PR TITLE
Avoid printing generic and duplicated "checking for QT" during ./configure

### PR DESCRIPTION
**Pull request by drizzt**
Print "checking for QT4" and "checking for QT5" instead

Reported by wumpus on irc

### CODE REVIEW

File: build-aux/m4/bitcoin_qt.m4 [hunk 1]
```diff
@@ -399,17 +399,17 @@ AC_DEFUN([_BITCOIN_QT_FIND_LIBS_WITH_PKGCONFIG],[
     qt4_modules="QtCore QtGui QtNetwork"
     BITCOIN_QT_CHECK([
       if test x$bitcoin_qt_want_version = xqt5 || ( test x$bitcoin_qt_want_version = xauto && test x$auto_priority_version = xqt5 ); then
-        PKG_CHECK_MODULES([QT], [$qt5_modules], [QT_INCLUDES="$QT_CFLAGS"; have_qt=yes],[have_qt=no])
```
Comment (laanwj) [inline build-aux/m4/bitcoin_qt.m4]: Does the first argument here influence anything else besides how it is printed in the logging?
https://developer.gnome.org/anjuta-build-tutorial/stable/library-autotools.html.en
Seems to imply `LIB_CFLAGS` and `LIB_LDFLAGS` get defined based on this parameter.

**Comment by fanquake** [comment]
OSX and linux builds failed.
```
  OBJCXX   qt/qt_bitcoin_qt-macdockiconhandler.o
  OBJCXX   qt/qt_bitcoin_qt-macnotificationhandler.o
In file included from qt/macdockiconhandler.mm:5:
./qt/macdockiconhandler.h:8:10: fatal error: 'QMainWindow' file not found
#include <QMainWindow>
         ^
In file included from qt/macnotificationhandler.mm:5:
./qt/macnotificationhandler.h:8:10: fatal error: 'QObject' file not found
#include <QObject>
         ^
1 error generated.
make[1]: *** [qt/qt_bitcoin_qt-macdockiconhandler.o] Error 1
make[1]: *** Waiting for unfinished jobs....
1 error generated.
make[1]: *** [qt/qt_bitcoin_qt-macnotificationhandler.o] Error 1
In file included from qt/bitcoin.cpp:9:
./qt/bitcoingui.h:14:10: fatal error: 'QLabel' file not found
#include <QLabel>
```
```
Making check in src
make[1]: Entering directory `/home/travis/build/bitcoin/bitcoin/build/bitcoin-x86_64-unknown-linux-gnu/src'
make[2]: Entering directory `/home/travis/build/bitcoin/bitcoin/build/bitcoin-x86_64-unknown-linux-gnu/src'
  CXX      qt/qt_bitcoin_qt-bitcoin.o
  CXX      qt/qt_libbitcoinqt_a-bantablemodel.o
In file included from qt/bitcoin.cpp:9:0:
qt/bitcoingui.h:14:18: fatal error: QLabel: No such file or directory
 #include <QLabel>
                  ^
compilation terminated.
make[2]: *** [qt/qt_bitcoin_qt-bitcoin.o] Error 1
make[2]: *** Waiting for unfinished jobs....
In file included from qt/bantablemodel.cpp:9:0:
qt/guiutil.h:12:23: fatal error: QHeaderView: No such file or directory
 #include <QHeaderView>
```

**Comment by paveljanik** [comment]
ACK https://github.com/bitcoin/bitcoin/pull/10549/commits/7222388bb24a2da47b0e3d47b67a91d7ab9495da

The build log diff is only:

```
-checking for QT... yes
+checking for QT5... yes
```

**Comment by laanwj** [comment]
Looks good to me now.
Ping @theuni

**Comment by fanquake** [comment]
utACK 7222388

**Comment by laanwj** [comment]
Tested ACK 7222388
```
checking for QT5... no
checking for QT4... no
configure: WARNING: Qt dependencies not found; bitcoin-qt frontend will not be built
```

**Comment by theuni** [comment]
post-merge utACK.

**Pull request merged**